src/Console/Commands/ConsumeCommand.php

```diff
@@ -83,17 +86,30 @@ public function handle()
               }
 
                 $bowlerConsumer = new Consumer(app(Connection::class), $handler->queueName, $exchangeName, $exchangeType, $bindingKeys, $passive, $durable, $autoDelete);
+
                 if ($deadLetterQueueName) {
 
                     // If configured as options and deadLetterExchangeName is not specified, default to deadLetterQueueName.
                     $deadLetterExchangeName = isset($deadLetterExchangeName) ? $deadLetterExchangeName : $deadLetterQueueName;
 
                     $bowlerConsumer->configureDeadLettering($deadLetterQueueName, $deadLetterExchangeName, $deadLetterExchangeType, $deadLetterRoutingKey, $messageTTL);
                 }
+
                 $bowlerConsumer->listenToQueue($handler->className, app(BowlerExceptionHandler::class));
             }
         }
 
         throw new UnregisteredQueueException('No registered queue found with name '.$queueName.'.');
     }
+
+    public function loadQueuesDefinitions()
+    {
+        $path = app_path().'/Messaging/queues.php';
+
+        if (!file_exists($path)) {
+            return $this->error('Queues definitions file not found. Please create it at '.$path);
```

Comment:
Add missing definition/configuration file bowler exception.

Reply:
Create a Bowler exception that specify that the file is missing.

Reply:
Exceptions should only be within the code and not at the command level. `error` takes care of responding properly in a CLI context.